src/Client.php

```diff
@@ -63,11 +63,21 @@ public function __construct(RemoteApi $remoteApi, ClientApi $clientApi, array $n
         $this->nodes = $nodes;
     }
 
+    /**
+     * Gets the remote API implementation.
+     *
+     * @return RemoteApi
```

Comment:
You can remove this php-doc information, because it's redundant to the return type